contracts/op-finality-gadget/src/exec/finality.rs

```diff
@@ -303,6 +305,29 @@ pub(crate) fn verify_finality_signature(
     Ok(())
 }
 
+/// `whitelist_forked_blocks` adds a set of forked blocks to the whitelist. These blocks will be skipped by FG
+/// (ie. treated as finalized) duringconsecutive quorum checks to unblock the OP derivation pipeline.
+pub fn whitelist_forked_blocks(
+    deps: DepsMut,
+    info: MessageInfo,
+    forked_blocks: Vec<(u64, u64)>,
+) -> Result<Response, ContractError> {
+    // Check caller is admin
+    check_admin(&deps, info)?;
+
+    // Check array is non-empty
+    if forked_blocks.is_empty() {
```

Comment:
should we also check the first element is less or equal to the second element in the tuple?